hack/build-dev-env.sh

```diff
@@ -13,10 +13,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+set -eu
 
 : "${NAMESPACE:=ingress-nginx}"
 echo "NAMESPACE is set to ${NAMESPACE}"
 
+
```

Comment:
what's this extra newline for? in the rest of the file groups are separated using single line

Reply:
Didn't realize a new line would be a deal breaker for contributing. I didn't see it in any style docs (there aren't any minus Go). I'll remove it.

Reply:
> Didn't realize a new line would be a deal breaker for contributing

@bobbytables we appreciate your contribution! Not a deal break at all, I should have tagged it with `nitpick` or something - just wanted at least per file style to be consistent.